src/components/PeopleItem/PeopleItem.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {TouchableOpacity, Text, View} from 'react-native';
+import getGlobalStyles from '~/helpers/GlobalStyles';
+import Colors from '../../helpers/Colors';
+import {getInitials} from '../../helpers/Functions';
+import Image from '../Image';
+import Icon from 'react-native-vector-icons/Fontisto';
+import getStyles from './PeopleItem.styles';
+
+const PeopleItem = ({avatar, firstName, lastName, gender, quote, address}) => {
+  const styles = getStyles();
+  const globalStyles = getGlobalStyles();
+
+  const renderAvatar = () => {
```

Comment:
Extract this into a separate `<Avatar />` component. The new component should either receive an image source or the initials. It will be reused in the details screen.